cdc_rsync_server/main.cc

```diff
@@ -82,7 +101,14 @@ int main(int argc, const char** argv) {
   cdc_ft::Log::Initialize(
       std::make_unique<cdc_ft::ConsoleLog>(cdc_ft::LogLevel::kWarning));
   cdc_ft::CdcRsyncServer server;
-  if (!server.CheckComponents(components)) {
+
+  const char* build_id = argv[4];
+  // If either cdc_server or cdc_client is dev version we can't compare build
+  // versions. So instead we need to compare file_size and modified_time.
+  if (((strcmp(build_id, DEV_BUILD_VERSION) == 0 ||
+        strcmp(BUILD_VERSION, DEV_BUILD_VERSION) == 0) &&
+       !server.CheckComponents(components)) ||
```

Comment:
Not necessarily for this CL, but since this logic will also be used in cdc_stream, I would suggest to put the build version logic into GameletComponent. I.e.
- GameletComponent::FromCommandLineArgs should expect "build_version comp1 size1 timestamp1 comp..." and return a struct containing the components and the build version.
- The != operator for components should implement this comparison logic.

Reply:
Make sense, will do it on separate review (maybe with adding build_version to executables)

Reply:
Actually it make sense to do in the current PR. So I'll add it here.

Reply:
OK, I'll wait for that change then.